packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -241,15 +243,15 @@ describe('Sayt Driver Plugin', () => {
     it('should get a response from Sayt client request method', () => {
       sendSaytApiRequest.resolves(response);
 
-      driver.fetchSaytData(query);
+      driver.fetchSaytData(fetchEvent);
 
-      expect(sendSaytApiRequest).to.be.calledWith(query);
+      expect(sendSaytApiRequest).to.be.calledWith(fetchEvent.detail);
```

Comment:
I would pull out `detail` into a `const` and use it here instead of drilling into `fetchEvent`. It gives you more control.